src/main/java/uk/gov/companieshouse/web/emergencyauthcodeweb/controller/eac/ListOfOfficersController.java

```diff
@@ -58,9 +58,13 @@ public String getListOfOfficers(@PathVariable String requestId, Model model, Htt
 
 
     @PostMapping
-    public String postListOfDirectors(@PathVariable String requestId,
-            @ModelAttribute("eacOfficer") @Valid EACOfficer eacOfficer, BindingResult bindingResult,
-            HttpServletRequest request, Model model) {
+    public String postListOfOfficers(@PathVariable String requestId,
+                                     @ModelAttribute("eacOfficer") @Valid EACOfficer eacOfficer,
+                                     BindingResult bindingResult,
+                                     HttpServletRequest request,
+                                     Model model) {
+
+        // If an officer is not selected, reload page and show error
```

Comment:
In my opinion your comments aren't really needed your code is well written and understandable at a glance. It's only opinion though so I will leave keeping or removing the comments up to you.